internal/trace/metrics.go

```diff
@@ -0,0 +1,69 @@
+package trace
+
+import (
+	"context"
+	"fmt"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+var keyInstance, _ = tag.NewKey("cloudsql_instance")
+
+var (
+	mLatencyMS = stats.Int64(
+		"/cloudsqlconn/latency",
+		"The latency in milliseconds per Dial",
+		stats.UnitMilliseconds,
+	)
+	latencyView = &view.View{
+		Name:        "/cloudsqlconn/dialer_latency",
+		Measure:     mLatencyMS,
+		Description: "The distribution of dialer latencies (ms)",
+		// Latency in buckets, e.g., >=0ms, >=100ms, etc.
+		Aggregation: view.Distribution(0, 100, 200, 300, 400, 500, 600, 800, 1000, 2000, 4000),
```

Comment:
I started with the latency aggregation in the OpenCensus quickstart for Go and adjusted the values to offer more granularity at the sub second range. But otherwise, there's no basis for this distribution.